### PR TITLE
Automated debian package building

### LINKED ISSUE #278: [Feature Request] Adding DEB package releases to the Github Releases
## What feature should be added to Tachidesk?
Not so much a feature added to Tachidesk, but to this repo. As the title says, adding the DEB packages from the already maintained Launchpad PPA to the stable binary releases on this repo.

##  Why/Project's Benefit/Existing Problem
Would make it significantly easier to put it on the [MPR.](https://mpr.hunterwittenborn.com/)

### PR DESCRIPTION
**Pull request by mahor1221**
Move wxs files to scripts/resources/msi/
Define icon's path inside windows-bundler.sh instead of hardcoding it in wxs file

Closes #278

Build test: https://github.com/mahor1221/Tachidesk-Server/releases

### CODE REVIEW

**Comment by AriaMoradi** [comment]
> Closes #278

It closes itself?!

File: scripts/resources/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tachidesk (${version}-1) unstable; urgency=medium
+ -- Mahor foruzesh <mahorforuzesh@pm.me>  Fri, 14 Jan 2022 00:00:00 +0000
```
Comment (AriaMoradi) [inline scripts/resources/debian/changelog]: maybe "Mahor Foruzesh"?

File: scripts/resources/debian/copyright [hunk 2]
```diff
@@ -0,0 +1,400 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: tachidesk
+Upstream-Contact: https://discord.gg/DDZdqZWaHA
+Source: https://github.com/Suwayomi/Tachidesk-Server
+
+Files: *
+Copyright: 2021 Contributors to the Suwayomi project
```
Comment (AriaMoradi) [inline scripts/resources/debian/copyright]: remove copyright year?

File: scripts/unix-bundler.sh [hunk 3]
```diff
@@ -74,6 +72,32 @@ archive_name=""
...
+  cd "$orig_dir/debian"
+  debuild -uc -us
+  deb_name="tachidesk_$release_ver-1_all.deb"
+  cd -
+
+  # revert back changes
+  rm -rf "$orig_dir/debian"
+  mv "$orig_dir" "$release_name"
+  mv "$orig_tar_gz" "$archive_name"
+
```
Comment (AriaMoradi) [inline scripts/unix-bundler.sh]: separate the deb related code from the tar builder code, I'd say define a new arch 'debian-x64' and deal with it separately and re-use `archive_name` in the last lines of the script.

**Review by AriaMoradi — CHANGES_REQUESTED** [review]
Move the windows script improvements into a separate PR

**Comment by mahor1221** [comment]
> > Closes #278
> 
> It closes itself?!

My bad. Grammar mistake
Just saying it's related to #278 and it can be closed after this PR get accepted

**Comment by mahor1221** [comment]
> > Closes #278
> 
> It closes itself?!

It seems it will really close that issue automatically. I didn't know that. just typed Closes and this happened. If you go to issues section you can see this PR is linked to the issue.

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: scripts/resources/debian/copyright [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+ This package is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <https://www.gnu.org/licenses/>
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License version 2 can be found in "/usr/share/common-licenses/GPL-2".
```
Comment (AriaMoradi) [inline scripts/resources/debian/copyright]: Why are you putting `tachidesk.png` and `tachidesk.png` under GPL 2+? shouldn't they be under MPL like the rest of the files?

**Review by AriaMoradi — CHANGES_REQUESTED** [review]

File: scripts/resources/debian/copyright (hunk 4 above)
Comment (mahor1221) [reply scripts/resources/debian/copyright]: Right. I simply copy paste this file from the ppa. I don't remember why I did this

**Pull request merged**